tests/test_api_lib_policy.py

```diff
@@ -1802,7 +1802,7 @@ def test_07_sign_response(self):
         new_response = sign_response(req, resp)
         jresult = json.loads(new_response.data)
         self.assertEqual(jresult.get("nonce"), "12345678")
-        self.assertEqual(jresult.get("signature"), "12286563956186851222763777429695584644324740652565581988587414544297432783523785532070055857619627542347973153465187816396752261730111152465977744264826200787508894389937562519240189748382275312824518790194963440380306202638365223197802490803597478514617706281242938294159192617108210623265084867605251009544178231020920999223970535034040822922068576537239452191722786808074622369000594984335557065543495017539992338002123756259365978375186660886179484370560915820484600337343618193501989005733515794341174572697366915377218917268995598532737466873248560512717129014931617597408250418461112752400750722207570913588830")
+        self.assertEqual(jresult.get("signature"), "11355158914966210201410734667484298031497086510917116878993822963793177737963323849914979806826759273431791474575057946263651613906587629736481370983420295626001055840803201448376203681672140726404056349423937599480275853513810616624349811159346536182220806878464577429106150903913526744093300868582898892977164229848617413618851794501457802670374543399415905458325601994002527427083792164898293507308423780001137468154518279116138010266341425663850327379848131113626641510715557748879427991785684858504631545256553961505159377600982900016536629720752767147086708626971940835730555782551222922985302674756190839458609")
```

Comment:
I understand that the keys of the response are now sorted for signature, so the signature changes but will stay the same?

Reply:
Yes, as long as there are the same keys/values in the dictionary. Before we
dumped the dictionary as a JSON-string without any order (which was fine
since the order did not change, but it wasn't guaranteed).
Now the the keys of the dictionary get sorted before the dump and thus the
dumped JSON-String should be the same regardless of the sorting of the
dictionary.